inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/adapter/RecommendationSpanRenderer.java

```diff
@@ -116,9 +121,10 @@ public void render(JCas aJcas, VDocument vdoc, AnnotatorState aState,
             // show only the confidence of the highest one
             for (AnnotationObject ao: token) {          
                 if (ao.getAnnotation() != null) {       
-                    if (isOverlapping(vspansWithoutRecommendations, ao.getOffset(), windowBegin)) {
-                        break;
-                    }
+//                    if (isOverlapping(vspansWithoutRecommendations, ao.getOffset(), windowBegin,
```

Comment:
I have a strong feeling this should be properly restored before merging the PR...